test/models/user_test.rb

```diff
@@ -49,4 +49,12 @@ def setup
     assert_not @user.valid?
   end
 
+  test "associated cheatsheets should be destroyed when a user is destroyed" do
+    @user.save
+    @user.cheatsheets.create!(title: "Test Header", topic: "test", content: "Lorem ipsum")
```

Comment:
I might create two just to make sure they're all destroyed, but there are people who'd suggest that's overkill.

Reply:
okay!! i do a lot of things that are probably overkill anyway. better safe than sorry!

Reply:
Anything worth doing, is worth overdoing (as one of my college friends used to suggest).